src/cls/Demo/ObjectScript.cls

```diff
@@ -0,0 +1,13 @@
+Class Demo.ObjectScript Extends %RegisteredObject
```

Comment:
Why Demo package is better than dc.python?
I especially introduced dc.python to minimize issues with ipm installation. if you have Demo package in the system you cannot install any other Demo. with dc.python package it is less probability to catch this.

Reply:
It's a bigger conversation, but Demo package is not enough. And where do we agree with PascalCase convention?
I raised the [conversation](https://community.intersystems.com/post/naming-convention-objectscript-packages-classes-and-package-manager-modules-names) before and it looks like community was OK to that. But I'm open for the discussion.

Reply:
It's a bigger conversation, but Demo package is not enough. And where do we agree with PascalCase convention?
I raised the [conversation](https://community.intersystems.com/post/naming-convention-objectscript-packages-classes-and-package-manager-modules-names) before and it looks like community was OK to that. But I'm open for the discussion.